src/utils/hooks/api/use-get-collateral-currencies.tsx

```diff
@@ -1,8 +1,7 @@
 import { CollateralCurrencyExt, getCollateralCurrencies } from '@interlay/interbtc-api';
```

Comment:
Wouldn't expect any regressions here, but we'll need to retest any features which are calling this hook.